src/core/assets.tf

```diff
@@ -0,0 +1,29 @@
+data "local_file" "tc" {
+  filename = "${path.module}/assets/tc.pdf"
+}
```

Comment:
can we use extended name, ie terms-condition?

Reply:
I'll change this approach using the same as contracts template

Reply:
would you like prefer to rename also the file?

Reply:
@antonioT90 yes, could you change the file name to extended version like terms-condition?

Reply:
renamed, InformativaPrivacy.pdf is ok?

Reply:
renamed into privacy-disclaimer.pdf